gnomad_qc/analyses/compute_pext.py

```diff
@@ -305,6 +322,24 @@ def main(args):
         default="v10",
         choices=["v7", "v10"],
     )
+    parser.add_argument(
+        "--min-samples",
+        help=(
+            "Minimum number of samples to include a tissue in the overall mean pext. "
+            "If keep-tissues-under-min-samples is not set, these tissues will also be "
+            "excluded from the individual tissue pext values."
+        ),
+        type=int,
+        default=50,
```

Comment:
100?

Reply:
Same comment about Bladder

Reply:
does there need to be additional logic based on today's ATGU/DSP meeting since we decided to keep bladder in the downloads but not display on the browser (because n samples < 100)?

Reply:
yup, just added